devel/geoipupdate/Portfile

```diff
@@ -1,37 +1,140 @@
+                        sha256  2f2e21ac8a9d523e88cbba4039441defc4a66bfaa78811c900a88fcf28729c4c \
+                        size    8702 \
+                    github.com/gofrs/flock \
+                        lock    v0.8.1 \
+                        rmd160  7ccfc1c50e85c62dab7e3ad2b0c7a0212c4f481c \
+                        sha256  6e35685e7796d4b36b90d4a908270e16868a44e06fcb890c1ec13d34cbb092ea \
+                        size    9059 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.1 \
+                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
+                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
+                        size    42171 \
+                    github.com/cenkalti/backoff \
+                        lock    v4.2.1 \
+                        rmd160  e4ac343410906a6a1ab0acd578ce034783b5ab42 \
+                        sha256  f30a8b3e79935ada4f425a1e19ba42863bc00e42a981d5b936f7555bd326651d \
+                        size    10399
 
-post-destroot {
-        #- move default conf to GeoIP.conf.default
-        file rename ${destroot}${prefix}/etc/GeoIP.conf ${destroot}${prefix}/etc/GeoIP.conf.default
+build.args          "-ldflags '" \
+                    "-X main.defaultConfigFile=${prefix}/etc/GeoIP.conf" \
+                    "-X main.defaultDatabaseDirectory=${prefix}/share/GeoIP" \
+                    "-X main.version=${version}'" \
+                    ./cmd/${name}
+
+set docs_path       ${worksrcpath}/doc
+set conf_path       ${worksrcpath}/conf
+
+test.run            yes
+test.cmd            ${worksrcpath}/${name}
+test.target         {}
+test.args           "-V"
+
+post-build {
+    foreach f [concat [glob -type f -dir ${docs_path} *.md] \
+        ${conf_path}/GeoIP.conf.default] {
+
+        # update paths before generating man pages
+        reinplace -q "s|CONFFILE|${prefix}/etc/GeoIP.conf|g" ${f}
+        reinplace -q "s|DATADIR|${prefix}/share/GeoIP|g" ${f}
+        reinplace -q "s|\$\$||g" ${f}
+    }
+    reinplace "s|#!/usr/bin/env perl|#!${prefix}/bin/perl5|" \
+        ${worksrcpath}/dev-bin/make-man-pages.pl
+    system -W ${docs_path} "${worksrcpath}/dev-bin/make-man-pages.pl ${docs_path}"
+}
+
+destroot {
+    # install bin file, man pages and config file
+    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin
+    file copy ${docs_path}/${name}.1 ${destroot}${prefix}/share/man/man1
+    file copy ${docs_path}/GeoIP.conf.5 ${destroot}${prefix}/share/man/man5
+    file copy ${conf_path}/GeoIP.conf.default ${destroot}${prefix}/etc
+
+    # install doc files
+    xinstall -d ${destroot}${prefix}/share/doc/${name}
+    xinstall -W ${worksrcpath} -m 0444 CHANGELOG.md LICENSE-APACHE LICENSE-MIT \
+        ${destroot}${prefix}/share/doc/${name}
+
+    # create folder to store database files
+    xinstall -d ${destroot}${prefix}/share/GeoIP
+    touch ${destroot}${prefix}/share/GeoIP/.placeholder
```

Comment:
I missed this when taking a look earlier, but the recommended way to do this is via `destroot.keepdirs`:

```
destroot.keepdirs   ${destroot}${prefix}/share/GeoIP

destroot {
...
```

If you do it this way, you can remove this line as you no longer need to create a placeholder file.